Makefile

```
@@ -1,12 +1,31 @@
+fetch:
+	tuist clean
+	tuist install
+gen:
+	tuist generate --no-open
 
-clean_xcode:
-	rm -rf ~/Library/Developer/Xcode/DerivedData/*
+sign: getig
+	fastlane sync
 	
+getig:
```

checkmake (0.2.2)

[warning] Target body for "getig" exceeds allowed length of 5 (15).

(maxbodylength)